### DISCUSSION TITLE
Automatically Deleting Old Images - Script

### DISCUSSION THREAD

**Discussion by danbayliss** [robmarkcole/HASS-Deepstack#197] [Show and tell]
Hi
Someone on the HA Facebook group helped me with a command to delete all deepstack images olders than 62 days except the latest.jpg image as I use them as local cameras. I've put it on here in case this is useful for anyone who wants to automate deleting old images. Add it to `configuration.yaml` and then you can use an automation to call it whenever you want

```# Shell command to delete old deepstack images
shell_command:
  delete_deepstack_images: find /config/www/deepstack -type f -mtime +62 | grep -v \._latest\.jpg | xargs -i rm "{}"```

**Comment by jodur** [comment]
You could use my custom integration for this as well.
[https://community.home-assistant.io/t/face-and-person-detection-with-deepstack-local-and-free/92041/1088?u=jodur](https://community.home-assistant.io/t/face-and-person-detection-with-deepstack-local-and-free/92041/1088?u=jodur)
